scenarios/kubernetes_e2e.py

```diff
@@ -442,6 +459,9 @@ def create_parser():
         '--test', default='true', help='If we need to set --test in e2e.go')
     parser.add_argument(
         '--up', default='true', help='If we need to set --up in e2e.go')
+    parser.add_argument(
+        '--multiple-federations', default=False, action='store_true',
```

Comment:
Make it part of the `--federation` logic?

No please stop adding gross new environment variables. The goal is to get rid of all these idiotic environment variables in e2e-runner.sh and delete e2e-runner.sh. Note how --build and --stage do not do any of this. They just pass the flag from the scenario to kubetest.

The goal is to move complex logic out of scripts and into kubetest.

The problem I am having is that this is creeping complexity into kuberenetes_e2e.py that should live in kubetest.